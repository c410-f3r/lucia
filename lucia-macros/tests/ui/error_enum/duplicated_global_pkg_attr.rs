#[lucia_macros::pkg(api(Foo), data_format(json))]
mod pkg {
  #[derive(Debug)]
  #[pkg::req_data]
  pub struct Req;

  #[pkg::req_data]
  struct Res;
}

fn main() {
}
