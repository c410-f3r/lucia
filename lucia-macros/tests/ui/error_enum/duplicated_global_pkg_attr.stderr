error: It is not possible to have more than one declaration of this `pkg` attribute in the same package.
 --> tests/ui/error_enum/duplicated_global_pkg_attr.rs:7:10
  |
7 |   #[pkg::req_data]
  |          ^^^^^^^^
